ext/search_kit/ang/crmSearchAdmin/crmSearchAdmin.component.js

```diff
@@ -156,6 +167,30 @@
       $scope.status = 'unsaved';
     }
 
+    // Generate the confirmation dialog
+    this.confirmSave = function() {
+      const targets = {}, data = {messages: []};
+      var newCount = 0;
```

Comment:
FYI I've been trying to get into the habit of using `let` instead of `var` when writing new code.

Reply:
Good point. `let` does sound more intuitive.